### DISCUSSION TITLE
ExplorerPatcher French version

### DISCUSSION THREAD

**Discussion by oOSachaOo** [valinet/ExplorerPatcher#2508] [Q&A]
Good morning,

Please forgive me for the grammatical and spelling mistakes, I use Google Translate to convert French to English!

I would like to ask if there is a French version of ExplorateurPatcher?

Because currently everything is in English and as you can imagine, I don't understand English!

Thanks for your help.

Sasha.

Bonjour,

Veuillez m'excuser pour les fautes grammaticales et d'orthographe, j'utilise Google traduction pour convertir le français en anglais !

J'aimerais demander s'il y a une version en français de ExplorateurPatcher ?

Car actuellement tout est en anglais et comme vous vous en doutez, je ne comprends pas l'anglais !

Merci pour votre aide.

Sacha.

**Comment by pyrates999** [comment]
Try the latest prerelease.  Supporting multiple languages has been started.

**Reply by Amrsatrio** [reply]
No, we don't have French yet in the latest prerelease. Though there is a PR in that L10N repository. I just don't have time yet to begin reviewing them.

**Reply by oOSachaOo** [reply]
Good evening,

Thank you for your responses, I understand, it is not always easy to respond to a multitude of requests for improvement of the ExplorerPatcher.

So I will wait for this to be available in future ExplorerPatcher updates

Good evening.
